nipoppy_cli/nipoppy/workflows/dataset_init.py

```diff
@@ -85,13 +122,138 @@ def run_main(self):
             FPATH_SAMPLE_MANIFEST, self.layout.fpath_manifest, log_level=logging.DEBUG
         )
 
+        if self.bids_source is not None:
+            self._init_manifest_from_bids_dataset()
+
+        if self.use_dalatad:
+
+            api.install(
+                dataset=dataset,
+                path=self.dpath_root / "code" / "templateflow",
```

Comment:
FYI the `self.layout` object has paths for the Nipoppy directory. For example `self.layout.dpath_proc` can be used instead of `self.dpath_root / 'code'`. See [docs](https://nipoppy.readthedocs.io/en/latest/schemas/index.html#layout-configuration) and [code](https://github.com/nipoppy/nipoppy/blob/8430fc18c21f7bba899ac1b9313fd923ba71dd17/nipoppy_cli/nipoppy/layout.py#L131) for more info

So that if we change the layout in the future it would not be super painful!

Reply:
thanks 
applied it everywhere I could

there is nothing to for the `code` folder yet, right?